test/file_existence.js

```diff
@@ -3,129 +3,117 @@ import fs from 'fs';
 import path from 'path';
 import glob from 'glob';
 
-import pkg  from './../package.json';
+import pkg from './../package.json';
 
 const dirs = pkg['h5bp-configs'].directories;
 
 const expectedFilesInArchiveDir = [
-    `${pkg.name}_v${pkg.version}.zip`
+  `${pkg.name}_v${pkg.version}.zip`,
 ];
 
 const expectedFilesInDistDir = [
 
-    '.editorconfig',
-    '.gitattributes',
-    '.gitignore',
-    '.htaccess',
-    '404.html',
-    'apple-touch-icon.png',
-    'browserconfig.xml',
+  '.editorconfig',
+  '.gitattributes',
+  '.gitignore',
+  '.htaccess',
+  '404.html',
+  'apple-touch-icon.png',
+  'browserconfig.xml',
 
-    'css/', // for directories, a `/` character
+  'css/', // for directories, a `/` character
             // should be included at the end
-        'css/main.css',
```

Comment:
Keep this formatting. This mirrors the file system and helps folks follow the file existence pattern.